### PR TITLE
Remove duplicate hotkeys code

### PR DESCRIPTION
**Pull request by EbenZhang**
Just code improvement.

### CODE REVIEW

**Comment by gerhardol** [comment]
Merging two duplications of code to one is an improvement.
(I added the second one in RevisionDiff, that was required after a much needed refactoring).

Compile is failing due to missing the following in GitExtensionsUserControl.cs
using System.Linq;

**Comment by EbenZhang** [comment]
fixed

File: GitUI/UserControls/RevisionGrid.cs [hunk 1]
```diff
@@ -3308,35 +3308,11 @@ public void GoToRef(string refName, bool showNoRevisionMsg)
...
-            var hotkey = GetHotkeyCommand(commandCode);
-            return hotkey == null ? Keys.None : hotkey.KeyData;
-        }
-
         internal Keys GetShortcutKeys(Commands cmd)
         {
             return GetShortcutKeys((int)cmd);
         }
 
-        /// <summary>
```
Comment (RussKie) [inline GitUI/UserControls/RevisionGrid.cs]: what about `internal Keys GetShortcutKeys(Commands cmd)`? both files have this method

Comment (EbenZhang) [reply GitUI/UserControls/RevisionGrid.cs]: they look the same, but of different types, one is `RevisionDiff.Commands`, whereas the other one is `RevisionGrid.Commands`

Comment (RussKie) [reply GitUI/UserControls/RevisionGrid.cs]: the cast the command down to int anyway

**Pull request merged**